chapters/fa/chapter2/2.mdx

```diff
@@ -0,0 +1,353 @@
+<FrameworkSwitchCourse {fw} />
```

Comment:
You should remove all the untranslated files - we only render on the website the chapters / sections that are translated